app/containers/UIKit/Image.tsx

```diff
@@ -30,7 +30,7 @@ export const Thumb = ({ element, size = 88 }: IThumb) => (
 export const Media = ({ element }: IImage) => {
 	const showAttachment = (attachment: IAttachment) => Navigation.navigate('AttachmentView', { attachment });
 	const imageUrl = element?.imageUrl ?? '';
-	return <ImageContainer file={{ image_url: imageUrl }} showAttachment={showAttachment} />;
+	return <ImageContainer onLongPress={null} file={{ image_url: imageUrl }} showAttachment={showAttachment} />;
```

Comment:
You probably won't be able to test it, so I'm going to left a comment here to make sure we don't forget to add it here.

Reply:
I think I have fixed the issue best check commit eff0fee6006a8dc5d4e397610a31a441dcc03fae